.config/CredScanSuppressions.json

```diff
@@ -0,0 +1,33 @@
+{
+    "tool": "Credential Scanner",
+    "suppressions": [
+        {
+            "file": "proxy_test.go",
+            "_justification": "sample login for testing"
+        },
+        {
+            "file": "pullsecret_test.go",
+            "_justification": "sample login for testing"
+        },
+        {
+            "file": "viper.go",
```

Comment:
not possible to bulk exclude vendor, *_test.go?

Reply:
Individual file listings only.